packages/react/src/components/cookieConsent/hooks/useCookieConsentEvents.ts

```diff
@@ -12,17 +13,13 @@ export type CookieConsentEventsProps = {
   onChange: (changeProps: CookieConsentChangeEvent) => void;
   onReady: () => void;
   onMonitorEvent: (changeProps: CookieConsentChangeEvent) => void;
-  submitEvent?: string;
+  submitEvent?: boolean;
 };
 export type CookieConsentEventsReturnType = () => void;
 
-export const defaultSubmitEvent = 'cookie-consent-changed';
-export const monitorEvent = 'hds-cookie-consent-unapproved-item-found';
-export const readyEvent = 'hds_cookieConsent_ready';
-
 export function useCookieConsentEvents(props: CookieConsentEventsProps): CookieConsentEventsReturnType {
   const { onChange, onReady, onMonitorEvent } = props;
-  const submitEvent = props.submitEvent || defaultSubmitEvent;
+  const submitEvent = props.submitEvent || false;
   const listenerDisposer: MutableRefObject<(() => void) | null> = useRef(null);
```

Comment:
When submitEvent was a string, this had to be settable with props. Now that it is a boolean, it should (practically must be) always `true`. There is no scenario where React needs to reload whole page when cookies change or not to listen events.